apollo-router/Cargo.toml

```diff
@@ -225,6 +225,10 @@ brotli = "3.3.4"
 zstd = "0.12.3"
 zstd-safe = "6.0.5"
 rand_core = "0.6.4"
+aws-sigv4 = "0.55.3"
+aws-credential-types = "0.55.3"
+aws-config = "0.55.3"
+aws-types = "0.55.3"
```

Comment:
We should probably highlight that AWS deps should always be the same version, or we could have type mismatches.

```suggestion
# note: AWS dependencies should always use the same version
aws-sigv4 = "0.55.3"
aws-credential-types = "0.55.3"
aws-config = "0.55.3"
aws-types = "0.55.3"
```

Reply:
Fixed in 9a6795ed